### PR TITLE
Windows Patches [Do not merge yet]

### PR DESCRIPTION
**Pull request by wargio**
Patches!!!

Removed some stupid warnings (like the malloc one)
Fixed the stacktrace

But i haven't tested it yet.

**so do not merge**

### CODE REVIEW

**Comment by Maijin** [comment]
Check https://github.com/radare/radare2/issues?q=is%3Aopen+is%3Aissue+label%3AWindows

**Comment by wargio** [comment]
Thanks

File: libr/include/sdb/sdb.h [hunk 1]
```diff
@@ -32,7 +32,7 @@ extern "C" {
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <io.h>
-extern void *__cdecl _aligned_malloc(size_t, size_t);
+extern __attribute__((dllimport)) void *__cdecl _aligned_malloc(size_t, size_t);
```
Comment (radare) [inline libr/include/sdb/sdb.h]: this change should be proposed in the sdb repo, not here

**Comment by radare** [comment]
please rebase

**Pull request closed without merging**